main.go

```diff
@@ -5,32 +5,64 @@ import (
 	"log"
 	"net/http"
 	"os"
+	"strings"
+	"text/template"
```

Comment:
Why use "text/template" rather than "html/template"? I think you'd get better escaping with the latter.

Reply:
Whoops.  Fixed in 8548d6bedbb4cac142d009fa81cc17abec79c879